### DISCUSSION TITLE
Help on sql server connection

### DISCUSSION THREAD

**Discussion by JOROZCOF206** [keycloak/keycloak#16563] [Q&A]
Hello.
I am trying to connect to a local database under the following description:

docker run --name mykeycloak -p 8081:8080 -d -e KC_DB=mssql -e KC_DB_USERNAME=sa -e KC_DB_PASSWORD=25800 -e KC_DB_URL='jdbc:sqlserver://MY IP:1433;database=KeyCloack' -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=2580 -e KC_TRANSACTION_XA_ENABLED=false quay.io/keycloak/keycloak:20.0.1 start-dev

But I get the following error:

![image](https://user-images.githubusercontent.com/83388263/213886333-81f4af2e-5983-4fdc-9a4d-92442edae05b.png)

Am I missing some parameter?

**Comment by filip-snapp** [comment]
I am stuck on the exact same issue for days now... @JOROZCOF206 any progress?

**Reply by filip-snapp** [reply]
Resolution for me was putting db password in single quotes. It contained special characters and that was causing issues...

**Reply by JOROZCOF206** [reply]
Hello @filip-snapp , same configuration on another machine and it works. I perform a test from another machine to my local DB and it works. I think the problem is the output of my machine to another host.
